libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -8,12 +8,14 @@
 #include <AP_SerialManager/AP_SerialManager.h>
 #include <AP_Math/AP_Math.h>
 #include <GCS_MAVLink/GCS.h>
+#include <AP_AHRS/AP_AHRS.h>
 
 #include "AP_DDS_Client.h"
 
 
 static constexpr uint16_t DELAY_TIME_TOPIC_MS = 10;
 static constexpr uint16_t DELAY_NAV_SAT_FIX_TOPIC_MS = 1000;
+static constexpr uint16_t DELAY_LOCAL_POSE_TOPIC_MS = 33;
```

Comment:
Is this a delay or is this a period??

Reply:
Its the minimum delay necessary for another message to be posted. In the loop it works as a period, yes. This corresponds to publishing the message at a frequency of ~30Hz.
About the variable name I'm following the standard naming of the other topics, as can be seen on DELAY_NAV_SAT_FIX_TOPIC_MS and DELAY_TIME_TOPIC_MS